jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/api/util/KubernetesResourceUtilTest.java

```diff
@@ -391,7 +393,7 @@ void mergeResources_whenBothDeploymentNonEmptySpec_thenShouldMergeBothObjects()
           .extracting(DeploymentSpec::getTemplate)
           .extracting(PodTemplateSpec::getSpec)
           .extracting(PodSpec::getContainers)
-          .asList()
+          .asInstanceOf(InstanceOfAssertFactories.list(Container.class))
           .singleElement(InstanceOfAssertFactories.type(Container.class))
```

Comment:
once you provide type information in `asInstanceOf` you can omit same type information in `singleElement`